virt/virtual_machines/cloning_vms/virt-cloning-vm-disk-into-new-datavolume.adoc

```diff
@@ -16,6 +16,11 @@ Cloning operations between different volume modes are supported, such as cloning
 However, you can only clone between different volume modes if they are of the `contentType: kubevirt`.
 ====
 
+[TIP]
+====
+When you globally enable preallocation, the Containerized Data Importer preallocates disk space to improve write performance during cloning. See xref:../../../virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc#virt-using-preallocation-for-datavolumes[Using preallocation for data volumes] for more information.
```

Comment:
Same comment applies to this.

Reply:
Same change as above and to the rest of the files except the smart-cloning assembly, from which the Tip was deleted.